docs/docsite/rst/playbook_guide/playbooks_filters.rst

```diff
@@ -930,7 +930,7 @@ To print out the ports from cluster1 in a comma separated string:
 
 .. note:: In the example above, quoting literals using backticks avoids escaping quotes and maintains readability.
 
-You can use YAML `single quote escaping <https://yaml.org/spec/current.html#id2534365>`_:
+You can use YAML `single quote escaping <https://yaml.org/spec/1.2.2/#single-quoted-style>`_:
```

Comment:
Since Ansible uses YAML 1.1 and not 1.2, it's better to link to https://yaml.org/spec/1.1/current.html.

I think the correct section is https://yaml.org/spec/1.1/current.html#id905860, since the below example shows `''` to quote a `'` inside `'`'s, which is also shown in that section of the YAML 1.1 spec.

```suggestion
You can use YAML `single quote escaping <https://yaml.org/spec/1.1/current.html#id905860>`_:
```